app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
```

Comment:
accepts list of what?
Also add inner type annotations to function elves_concert, feast_of_the_dwarves